composer.json

```diff
@@ -1,39 +1,39 @@
 {
-  "name": "beatlabs/pyr",
-  "description": "A Prometheus bridge for Lumen",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Beat"
+    "name": "beatlabs/pyr",
```

Comment:
@marcos-sandim , I think you said we were going to change the package vendor name to arquivei like `arquivei/pyr`. Is that right?

Reply:
I dit it, on the other open PR: `arquivei/laravel-prometheus-exporter`